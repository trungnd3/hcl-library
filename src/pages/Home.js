import Centered from '../components/UI/Centered';

const Home = () => {
  return (
    <Centered>
      <h1>Welcome libraian!</h1>
    </Centered>
  );
};

export default Home;
