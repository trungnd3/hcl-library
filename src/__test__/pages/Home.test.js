import React from 'react';
import Home from '../../pages/Home';
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';

describe('Home component using RTL', () => {
  test('render snapshot', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('render Home', () => {
    render(<Home />);
    const homeText = screen.getByText('Welcome member!');
    expect(homeText).toBeInTheDocument();
  });
});
