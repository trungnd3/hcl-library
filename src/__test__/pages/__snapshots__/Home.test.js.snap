// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component using RTL render snapshot 1`] = `
<div
  className="mt-5 text-center"
>
  <h1>
    Welcome libraian!
  </h1>
</div>
`;
